File: pset6/mario/more/mario.py
from cs50 import get_int

while True:
    a = get_int("Height: ")
    if a > 0 and a!=24:
        break
height = a

for i in range(height):
    for a in range(height - i - 1):
        print(" ", end = "")
    for b in range(i + 1):
        print("#", end = "")
    print("  ", end = "")
    for c in range(i + 1):
        print("#", end = "")
    print()

File: pset1/mario/less/mario.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    int n;

    do
    {
        n = get_int("Height: ");
    }
    while (n < 0 || n > 23);

    for (int i = 0; i < n; i++)
    {
        for (int b = 0; b < n - 1 - i; b++)
        {
            printf(" ");
        }
        for (int a = 0; a < i + 2; a++)
        {
            printf("#");
        }
        printf("\n");
    }
}